networkx/readwrite/tests/test_graph6.py

```diff
@@ -109,9 +109,59 @@ def test_write_path(self):
             assert f.read() == b">>graph6<<?\n"
 
     def test_relabeling(self):
-        G = nx.Graph([(0, 1)])
-        assert g6.to_graph6_bytes(G) == b">>graph6<<A_\n"
-        G = nx.Graph([(1, 2)])
-        assert g6.to_graph6_bytes(G) == b">>graph6<<A_\n"
-        G = nx.Graph([(1, 42)])
-        assert g6.to_graph6_bytes(G) == b">>graph6<<A_\n"
+        for edge in [(0, 1), (1, 2), (1, 42)]:
+            G = nx.Graph([edge])
+            f = BytesIO()
+            nx.write_graph6(G, f)
+            f.seek(0)
+            assert f.read() == b">>graph6<<A_\n"
```

Comment:
+1 for this change, but it'd be a slight improvement to parametrize the test instead of use a forloop, e.g.

```python
@pytest.mark.parametrize("edge", ((0, 1), (1, 2), (1, 42)))
def test_relabeling(self, edge):
    G = nx.Graph([edge])
    f = BytesIO()
    nx.write_graph6(G, f)
    f.seek(0)
    assert f.read() == b">>graph6<<A_\n"
```